rtf_vis_tool/src/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+
+ReactDOM.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+  document.getElementById('root')
+);
```

Comment:
This React component is imported into index.html. So the way React is reading and displaying these files is:

1. first reads `index.html` (which has `index.js` imported into it)
2. then reads `index.js` (which has `App.js` imported into it)
3. then reads `App.js` (which is what is finally shown on the screen with text: "boilerplate App.js body")